c-dependencies/js-compute-runtime/builtins/crypto.cpp

```diff
@@ -44,7 +45,7 @@ bool Crypto::get_random_values(JSContext *cx, unsigned argc, JS::Value *vp) {
   JS::AutoCheckCannotGC noGC(cx);
   bool is_shared;
   void *buffer = JS_GetArrayBufferViewData(typed_array, &is_shared, noGC);
-  arc4random_buf(buffer, byte_length);
+  random_get((int32_t)buffer, byte_length);
```

Comment:
Isn't this going to be pretty slow? Should we pull over a PRNG and seed it from `random_get` when initializing?